### PR TITLE
Adding 23.2.0 scripts 

### PR DESCRIPTION
**Pull request by IshaanDesai45**
Adding 23.2.0 scripts into the dockerfiles folder of singleInstance database with required changes 

Issues - #2611

### CODE REVIEW

**Review by gvenzl — DISMISSED** [review]

**Comment by gvenzl** [comment]
Looks good to me but there are linter issues, @IshaanDesai45.

**Comment by gvenzl** [comment]
Hi @spavlusieva, the Linter process fails although it only produces infos and warnings for legitimate cases. How can we get it to accept these situations in the scripts going forward, please advise.

**Comment by IshaanDesai45** [comment]
@gvenzl I have removed most of the linting errors and disabled linting checks for some. We can merge this PR now. 
One request is if possible lets do a squash merge of this PR.

File: OracleDatabase/SingleInstance/dockerfiles/23.2.0/runOracle.sh [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+      mkdir -p "$ORACLE_BASE"/oradata/dbconfig/"$ORACLE_SID"/
+   fi;
+
+   mv "$ORACLE_BASE_CONFIG"/dbs/spfile"$ORACLE_SID".ora "$ORACLE_BASE"/oradata/dbconfig/"$ORACLE_SID"/
+   mv "$ORACLE_BASE_CONFIG"/dbs/orapw"$ORACLE_SID" "$ORACLE_BASE"/oradata/dbconfig/"$ORACLE_SID"/
+   mv "$ORACLE_HOME"/network/admin/sqlnet.ora "$ORACLE_BASE"/oradata/dbconfig/"$ORACLE_SID"/
+   mv "$ORACLE_HOME"/network/admin/listener.ora "$ORACLE_BASE"/oradata/dbconfig/"$ORACLE_SID"/
+   mv "$ORACLE_HOME"/network/admin/tnsnames.ora "$ORACLE_BASE"/oradata/dbconfig/"$ORACLE_SID"/
+   # shellcheck disable=SC2144
+   if [ -a "$ORACLE_HOME"/install/.docker_* ]; then
```
Comment (gvenzl) [inline OracleDatabase/SingleInstance/dockerfiles/23.2.0/runOracle.sh:26]: This shellcheck should not be disabled. This if statement will fail if there are more than 1 `.docker_*` files present:

```
gvenzl$ touch .docker_1
gvenzl$ touch .docker_2
gvenzl$ touch .docker_3
gvenzl$ if [ -a .docker_* ]; then
>   echo true;
> fi;
-bash: [: too many arguments
gvenzl$
```

File: OracleDatabase/SingleInstance/dockerfiles/23.2.0/runUserScripts.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+if [ -d "$SCRIPTS_ROOT" ] && [ -n "$(ls -A "$SCRIPTS_ROOT")" ]; then
+
+  echo "";
+  echo "Executing user defined scripts"
+
+  for f in "$SCRIPTS_ROOT"/*; do
+      case "$f" in
+          *.sh)     
+          echo "$0: running $f"; 
+          # shellcheck disable=SC1090
```
Comment (gvenzl) [inline OracleDatabase/SingleInstance/dockerfiles/23.2.0/runUserScripts.sh:31]: This should be `# shellcheck source=/dev/null` instead

**Review by gvenzl — CHANGES_REQUESTED** [review]
Please see comments

File: OracleDatabase/SingleInstance/dockerfiles/23.2.0/runOracle.sh (hunk 1 above)
Comment (IshaanDesai45) [reply OracleDatabase/SingleInstance/dockerfiles/23.2.0/runOracle.sh]: resolving this with 
```
for file in "$ORACLE_HOME"/install/.docker_*
   do 
      if [ -e "$file" ]; then 
         mv "$file" "$ORACLE_BASE"/oradata/dbconfig/"$ORACLE_SID"/
      fi 
   done
```

Comment (IshaanDesai45) [reply OracleDatabase/SingleInstance/dockerfiles/23.2.0/runOracle.sh]: Better replaced the for loop with find command alternative
`find "$ORACLE_HOME"/install/ -name ".docker_*"  -exec  mv {} "$ORACLE_BASE"/oradata/dbconfig/"$ORACLE_SID"/  \;`

Comment (gvenzl) [reply OracleDatabase/SingleInstance/dockerfiles/23.2.0/runOracle.sh]: Awesome, that's even better!

**Review by gvenzl — APPROVED** [review]

**Pull request merged**